general/version.hpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-443211. All Rights
+// reserved. See file COPYRIGHT for details.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability see http://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the GNU Lesser General Public License (as published by the Free
+// Software Foundation) version 2.1 dated February 1999.
+
+#ifndef MFEM_VERSION_H
+#define MFEM_VERSION_H
+
+namespace mfem
+{
+
+int get_version();
```

Comment:
Switch to `GetVersion()` instead of `get_version()`? Seems more mfem-y

Reply:
We do use "snake_case" for some functions, e.g. mfem_error(), and some classes, e.g. socketstream. I'm OK with both.

Reply:
Changing to camel case in my upcoming update.